src/wizard/interactive_build.jl

```diff
@@ -503,7 +504,10 @@ function step5c(state::WizardState)
                 verbose=false,
                 tee_stream=state.outs
             )
-
+            if prefix == nothing
```

Comment:
It would also be nice to have a comment here describing what `prefix == nothing` means.